sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -0,0 +1,3270 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-03-13",
+    "title": "IotHub Gateway Service APIs"
+  },
+  "host": "fully-qualified-iothubname.azure-devices.net",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/configurations/{id}": {
+      "get": {
+        "summary": "Retrieve a configuration for Iot Hub devices and modules by it identifier.",
```

Comment:
their identifier instead of "it"

Reply:
mmm, I'd say "the identifier" or if you want possessive "its identifier".

"their" is reserved for people, not things.

Reply:
"its identifier" works for me, good catch

Reply:
good to know ! "its" then